WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/WatchDog.java

```diff
@@ -22,8 +23,18 @@
 import nl.tudelft.watchdog.intellij.util.WatchDogUtils;
 import org.jetbrains.annotations.NotNull;
 
+import javax.swing.*;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.awt.event.WindowFocusListener;
+import java.io.File;
+
```

Comment:
Rename to WatchDogStartUp or similar. Document 'strange' start behavior in comment?